ta/os_test/ta_entry.c

```diff
@@ -92,6 +92,9 @@ TEE_Result TA_InvokeCommandEntryPoint(void *pSessionContext,
 	case TA_OS_TEST_CMD_TA2TA_MEMREF:
 		return ta_entry_ta2ta_memref(nParamTypes, pParams);
 
+	case TA_OS_TEST_CMD_TA2TA_MEMREF_SIZE0:
+		return ta_entry_ta2ta_memref_empty(nParamTypes, pParams);
```

Comment:
How about naming this function `ta_entry_ta2ta_memref_size0()` to match the TA command ID?

Reply:
Suggestion applied in https://github.com/OP-TEE/optee_test/pull/708/commits/63786dc212cdbd692df3e47b6429a8280554374c. Thanks.